mlir/lib/Dialect/Rock/Transforms/AlignTiling.cpp

```diff
@@ -129,57 +122,127 @@ static Value applyTransforms(PatternRewriter &b, ThreadwiseWriteAllOp storeOp,
   moveTransformsBefore(b, src);
 
   // 2.1. apply transform chain from output
-  src = transformAccordingly(b, src, storeOp.getDest());
+  auto extraViews = extractVector(storeOp.getExtraViews());
+  extraViews.insert(extraViews.end(), relativeViewsOnStore.begin(),
+                    relativeViewsOnStore.end());
 
   // 2.2. load into registers
-  b.create<ThreadwiseReadIntoOp>(loc, src, alloc, storeOp.getExtraViews(),
+  b.create<ThreadwiseReadIntoOp>(loc, src, alloc, b.getArrayAttr(extraViews),
```

Comment:
Can we not do this? `extraViews` should be the TID->GEMM maps? But it's not _wrong_.

Reply:
yes we could... I found it easier to read the IR this way. Any downside ? 
(I just needed this to speed up my debugging while developing this... but can use TransformOps if that effort is worthwhile spending..)

Reply:
I think it's easier to conceptually understand when there's a sequence of `rock.transform` calls? 

This is more because of the theory that we're always aligning the writes out to GEMM space, so `extraViews` is a minimal thing that `gridwise_gemm_to_blockwise` uses to translate from `{bid, tid, i)` to GEMM coordinates, and then the transforms on the value take you the rest of the way there.

It's not a hard requirement, but ... I kinda like it the way we had it before.

Reply:
I ll have a go tomorrow to see but we might wanna codify an op verifier if thats desired to reduce ambiguity in the future

Reply:
(Though I still think its easier to read it left to right compared to interleaved transform chains in fusion scenarios -- so its subjective)

Reply:
now done!